doc/source/reference/ccallback.rst

```diff
@@ -1,30 +0,0 @@
-============================
-Low-level callback functions
```

Comment:
I think moving all this into the `LowLevelCallable` documentation turned out fine.